sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -5145,10 +5145,16 @@ class AstBuilder extends DataTypeAstBuilder
    */
   override def visitDescribeRelation(ctx: DescribeRelationContext): LogicalPlan = withOrigin(ctx) {
```

Comment:
We can move this entire method to `SparkSqlAstBuilder`, so that it can access `DescribeTableJsonCommand` directly, no need to create `DescribeRelationJson`